src/main/java/com/linuxforhealth/connect/builder/LinuxForHealthRouteBuilder.java

```
@@ -86,6 +86,7 @@ public void configure() {
             LinuxForHealthMessage msg = new LinuxForHealthMessage(exchange);
             msg.setError(exception.getMessage());
             exchange.getIn().setBody(msg.toString());
+            exception.printStackTrace();//TODO remove
```

Line 89 can probably be removed now.

---

@ccorley @dixonwhitmire  Agree, this was intended to be removed. But I do want to point out that I ran into a few scenarios where it would have been nearly impossible to debug the cause for an error because we swallow the root target exception and only display the last message from the thrown exception. I'll open a separate issue on this.